README.md

```diff
@@ -39,6 +39,15 @@ With the first Gradle invocation, Gradle will take the value of the `MAPBOX_ACCE
 ```xml
 <string name="access_token">PASTE_YOUR_TOKEN_HERE</string>
 ```
+##### Build variants
+
+The app has a two build flavors; `global` and `china`. The `global` flavor has the full list of standard examples, whereas the `china` flavor has China-specific examples, which use [the Mapbox China Plugin for Android](https://docs.mapbox.com/android/plugins/overview/china/). The plugin requires a special China Mapbox access token. [Contact our sales team to start the process of receiving this special access token](https://www.mapbox.com/contact/sales/).
+
+You might see China-related examples when you locally run the app on your device. This is because Android Studio is set on the `china` flavor. Find and click on the "Build Variants" tab in the lower left hand corner of the Android Studio window. Then choose `globalDebug` to change the flavor to the `global` flavor. Re-run the app on your device to see the "regular" demo app examples.
```

Comment:
Maybe we can mention that examples will probably crash if you've built with the china flavor but are not in China?

ref. https://github.com/mapbox/mapbox-android-demo/issues/914#issuecomment-439298105

Reply:
If the `china` flavor is built and a valid China access token is being used, the examples won't crash 🤔

Reply:
> and a valid China access token is being used
Yes, but in cases where you build the app from source it is likely you will not have set a valid China access token. And to clarify, the crash won't happen until you click on an example.

Reply:
>Yes, but in cases where you build the app from source it is likely you will not have set a valid China access token.

Not sure what you mean @riastrad .  Or rather, I'm unclear on what information you believe is still missing from this pr's additions.

Reply:
Woah - sorry this dropped off my radar @langsmith. I tried to recreate the issue I was referencing back in March, and could not do it. I think we can chalk this one up to "user error" on my part. Other than my unfounded concerns, this PR looks good and we should merge 👍